modules/core/tests/server/services/influx.server.service.tests.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+require('should');
+
+var path = require('path'),
+    influxService = require(path.resolve('./modules/core/server/services/influx.server.service')),
+    config = require(path.resolve('./config/config'));
+
+describe('Service: influx', function() {
+
+  context('InfluxDB disabled', function () {
+    var originalInfluxSettings;
+
+    before(function () {
+      originalInfluxSettings = config.influxdb.enabled;
+      config.influxdb.enabled = false;
+    });
+
+    after(function () {
+      config.influxdb.enabled = originalInfluxSettings;
+    });
+
+    it('Getting client returns error if no InfluxDB configured', function(done) {
+      influxService.getClient(function(err) {
+        err.message.should.equal('No InfluxDB configured.');
```

Comment:
as we already discussed elsewhere, the _should_ assertion probably needs to be wrapped in `try catch` in order to catch the failing assertion. Otherwise it will timeout.